CoreFoundation/CMakeLists.txt

```diff
@@ -41,8 +41,18 @@ else()
   set(FRAMEWORK_LIBRARY_TYPE STATIC)
 endif()
 
+set(cf_swift_runtime_abi_flags)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  set(cf_swift_runtime_abi_flags -fcf-runtime-abi=swift)
+endif()
+
+set(cf_darwin_only_mach_port_sources)
+if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  set(cf_darwin_only_mach_port_sources RunLoop.subproj/CFMachPort.c;RunLoop.subproj/CFMachPort_Lifetime.c;RunLoop.subproj/CFMessagePort.c)
+endif()
```

Comment:
UGH, I really wish that `target_sources` worked :-(.  Would you be okay with always building this code, and just adding a `#if defined(__MACH__)` around the source file?